tests/README.md

```diff
@@ -31,6 +31,309 @@ TC_PATHS="tests/kafkatest/tests/streams tests/kafkatest/tests/tools" bash tests/
   - The docker containers are named knode01, knode02 etc.
    These nodes can't be used for any other purpose.
 
+Examining CI run
+----------------
+* Set BUILD_ID is travis ci's build id. E.g. build id is 169519874 for the following build
```

Comment:
I'm guessing this is a chicken & egg situation, but we probably want to replace this eventually with an example ID that actually ran some tests :) Unless something went wrong when I ran the commands with this build ID? The docs here suggest I'll get output for tests, but all the test counts are zero for me w/ this ID.